apps/yarn.lock

```diff
@@ -1561,8 +1561,7 @@
     to-fast-properties "^2.0.0"
 
 "@cdo/interpreted@link:../dashboard/config/libraries":
-  version "0.0.0"
-  uid ""
+  version "0.1.0"
```

Comment:
Just curious, why is this version number going up when rolling something backwards?